src/components/DefaultLayout/DefaultLayout.js

```diff
@@ -53,6 +58,10 @@ class DefaultLayout extends Component {
             >
               <>
                 {!hideHeader && <Header />}
+                <Notifications
+                  type
+                />
+                <NotificationButtons />
```

Comment:
why did you commit `<NotificationButtons />` component ?

Reply:
i did it because there was no example how to use it for other team members.
I was planning to remove it after using Notifications in other components :)

Reply:
it's ok for testing, but you shouldn't commit such type of component

Reply:
ok, my bad